python/paddle/distributed/checkpoint/save_state_dict.py

```diff
@@ -156,9 +147,11 @@ def save_state_dict(
         state_dict(Dict[str, paddle.Tensor]): The state_dict to save.
         path(str): The directory to save state_dict.
         process_group(paddle.distributed.collective.Group): ProcessGroup to be used for cross-rank synchronization. Use the default process group which contains all cards.
-        coordinator_rank(int): The rank used to save non distributed values. Rank0 is used by default.
+        coordinator_rank(int): The rank used to save non distributed values. Rank 0 is used by default.
+        unique_id(int): The unique id of ckeckpoint, used to distinguish between different checkpoint versions. Default is None, in which case the id 0 when save for the first time and increased by 1 each time when calling save_state_dict in the same path.
         async_save(bool): Async save the state_dict, default is False.
 
+        Note: If there is already checkpoint in
```

Comment:
```suggestion
        Note:
            If there is already checkpoint in

```
![image](https://github.com/user-attachments/assets/483b197a-0b2b-45dc-a508-7fa60b0f746b)